types/open-location-code/index.d.ts

```diff
@@ -118,7 +116,7 @@ declare class OpenLocationCode {
      *     area of the code.
      * @throws {@link Exception} If the code is not valid.
      */
-    static decode(code: string): OpenLocationCode.CodeArea;
+    static decode(code: string): CodeArea;
```

Comment:
I tested this PR, and the `static` here should be removed as well. 
Currently it throws this error:
<img width="938" alt="image" src="https://github.com/user-attachments/assets/1a028d41-ec22-4255-9f6d-c14a838b7b38" />

The OpenLocationCode class has to be instantiated first to call member method upon it like this: 
```
const olc=new OpenLocationCode();
olc.encode(...);
```
Unbelievably - they have it wrong in their readme, but [the code speaks](https://github.com/google/open-location-code/blob/main/js/src/openlocationcode.js#L311) for itself :)

Reply:
Hm, but this is declared just like all of the other functions above like `isFull` too, no?

Reply:
yes, i think all of the functions are not static. They are declared on the prototype as you see in the source code.

Reply:
But they're not on `OpenLocationCode.prototype`, they're on `OpenLocationCode`